pkg/webhook/policy.go

```diff
@@ -599,7 +605,7 @@ func (h *validationHandler) reviewRequest(ctx context.Context, req *admission.Re
 }
 
 func (h *validationHandler) review(ctx context.Context, review interface{}, trace bool, dump bool) (*rtypes.Responses, error) {
-	resp, err := h.opa.Review(ctx, review, drivers.Tracing(trace), drivers.Stats(*logStatsAdmission))
+	resp, err := h.opa.Review(ctx, review, reviews.SourceEP(util.WebhookEnforcementPoint), reviews.Tracing(trace), reviews.Stats(*logStatsAdmission), reviews.SourceEP(util.WebhookEnforcementPoint))
```

Comment:
why does reviews.SourceEP(util.WebhookEnforcementPoint) need to be passed in twice

Reply:
Fixed it. It doesn't need to be passed twice.